tests/app/test_application.py

```diff
@@ -4,7 +4,7 @@
 class TestApplication(BaseApplicationTest):
     def test_index(self):
         response = self.client.get('/')
-        assert 200 == response.status_code
+        assert 401 == response.status_code
```

Comment:
@TheDoubleK Probably have to disable authentication for the tests. And I'm not sure how we're going to deal with testing this app on production—don't want them editing real services.